src/components/common/IntersectionNavbar.tsx

```diff
@@ -58,15 +58,15 @@ export default function IntersectionNavbar() {
       const sectionId = href.split('/#')[1];
       const target = document.getElementById(sectionId);
       if (target) {
-        target.scrollIntoView({ behavior: 'smooth', block: 'start' });
```

Comment:
When a link is clicked, the former code scrolls to the section as a ref, whilst the incoming code scrolls only to the section header as a ref (e.g. Splash, About, etc).

See difference:
Former
![image](https://github.com/user-attachments/assets/f251b1f4-8d71-4aa6-b417-4f0cc5baa70d)

vs

Incoming
![image](https://github.com/user-attachments/assets/430fb32d-bc74-4c4c-981e-1613fcd9a315)